### PR TITLE
chore(deps): bump dagstore 

### PR DESCRIPTION
**Pull request by walldiss**
Updating dagstore should fix the issue with panic of mmap on bitswap reads.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -80,6 +80,8 @@ require (
 	google.golang.org/protobuf v1.31.0
 )
 
+require github.com/ipfs/go-ipfs-exchange-offline v0.3.0
```
Comment (Wondertan) [inline go.mod:83]: Idk why go does it for you

Comment (walldiss) [reply go.mod]: There as a staged test. fixed now

**Review by Wondertan — APPROVED** [review]

**Comment by distractedm1nd** [comment]
We have to bump dagstore again so don't approve this PR until vlad is awake (its on automerge)

**Comment by walldiss** [comment]
It is ready now.

**Review by Wondertan — APPROVED** [review]

**Review by distractedm1nd — APPROVED** [review]

**Pull request merged**